mtp_api/apps/mtp_auth/migrations/0021_alter_role_other_groups.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.2.20 on 2025-03-13 15:17
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
did this migration come about cos we didn't `makemigrations` in the last batch of django updates?
cos `auth.0012…` happened in [django 3.1]/9https://docs.djangoproject.com/en/5.1/releases/3.1/#abstractuser-first-name-max-length-increased-to-150)

Reply:
I'm not sure, but possibly we didn't run `makemigrations` before? 🤷‍♂️